src/coreclr/jit/compiler.hpp

```diff
@@ -101,52 +101,30 @@ inline T genFindLowestBit(T value)
 /*****************************************************************************
  *
  *  Return the highest bit that is set (that is, a mask that includes just the highest bit).
- *  TODO-ARM64-Throughput: we should convert these to use the _BitScanReverse() / _BitScanReverse64()
- *  compiler intrinsics, but our CRT header file intrin.h doesn't define these for ARM64 yet.
  */
-
 inline unsigned int genFindHighestBit(unsigned int mask)
```

Comment:
Same here. LGTM but this is effectively using `LeadingZeroCount` (and then masking out everything but the highest bit) but without strictly handling some of the potential edge cases.

It might be worthwhile to centralize these somewhere (maybe as an "up for grabs" work item)